src/components/Portfolio/PortfolioList/PortfolioList.tsx

```diff
@@ -46,15 +46,15 @@ const PortfolioList = () => {
         console.log(response.data);
       })
       .catch((error) => {
-        console.log(error);
+        console.log("error");
```

Comment:
Why?

Reply:
Removed  console.log(response.data);

Reply:
removed quotes from error. Not sure why it was put in quotations.